movevm/liquidity-provider/README.md

```diff
@@ -8,64 +8,72 @@ This way, you should have:
 - initiad binary;
 - mnemonic with some INIT tokens on initiation-2 network.
 
-#### 2. Configure deployment
-
-Navigate to `Move.toml` and open it in your editor of choice. You are interested in the
-section `[addresses]`. `me` and `recipient` are filled with placeholder (`_`) addresses,
-so you will have to fill them up. You can use this easy snippet to generate hexadecimal
-addresses from keys stored in your initiad keychain:
+#### 2. Get your account address in hexademical format
 
 ```bash
 NAME="<name of your key>"; echo "0x$(initiad keys parse "$(initiad keys show "$NAME" --output json | jq -r '.address')" --output json | jq -r '.bytes' | tr '[:upper:]' '[:lower:]')"
 ```
 
-First, fill `me` to the address of your own account. For `recipient`, you can create a new
-empty account and use it's address.
-
 #### 3. Build module
 
-It is as easy as `initiad move build`.
+It is as easy as
+
+```bash
+initiad move build --named-addresses "me=<your hex address from step 2>"
+```
 
 #### 4. Deploy module
 
-It is also easy, sign your normal Cosmos SDK transaction:
+It is also as easy as
 
 ```bash
 initiad move deploy --path "$(pwd)" --upgrade-policy COMPATIBLE --from <name of your key> --gas auto --gas-adjustment 1.5 --gas-prices 0.025uinit --node https://rpc.initiation-2.initia.xyz:443 --chain-id initiation-2
 ```
 
-#### 5. Determine address of module object
+#### 5. Instantiate a new Liquidity Provider object
+
+Method `create_liquidity_provider` has several arguments:
 
-Open module upload transaction in block explorer, for example take a look at
-[this one](https://scan.testnet.initia.xyz/initiation-1/txs/7B408B00337E840D0AF2BB89615CEEFFD73A28458D1BD31185418909FAF37BDB).
-Look for event log with `type_tag` equal to `0x1::object::CreateEvent`.
-Inside this event there is a JSON, containing a field `object` with the
-address of our new module object. This address is where INIT tokens are expected
-to be deposited to.
+- string:\<name\> Name for a visual reference, e.g. "testnet_uinit"
+- option\<address\>:\<address\> Priviliged account with a permission to withdraw any coins from the module object, will be set to `account` if omitted
```

Comment:
You know, the more I think about it, the more I regret making this field optional. Maybe just make it mandatory, what do you think?